holoviews/core/data/iris.py

```diff
@@ -0,0 +1,248 @@
+from __future__ import absolute_import
+
+import datetime
+from itertools import product
+
+import iris
+import numpy as np
+
+from .interface import Interface
+from .grid import GridInterface
+from ..ndmapping import (NdMapping, item_check, sorted_context)
+from ..spaces import HoloMap, DynamicMap
+from .. import util
+
+from iris.util import guess_coord_axis
+from holoviews.core.dimension import Dimension
+
+
```

Comment:
The three functions below are fine. I just hope we don't accumulate *too* many functions floating outside the class itself.